crates/uniswapv3pool/src/pool_calcs.rs

```diff
@@ -0,0 +1,17 @@
+use uniswap_v3_sdk::prelude::FeeAmount;
+
+pub fn fee_to_float(fee: FeeAmount) -> f32 {
```

Comment:
Why does this need to be a float?

Reply:
Because the result can be 0.01, for instance. Though the current usage of the result is as a string, it is likely that the result will be used in calculations of the cost of doing a price change.